contributing/workflow.qmd

```diff
@@ -649,14 +654,104 @@ Cookbook that have that link, and then you can do a search-replace to
 update them all at once.
 
 ## Cookbook Editing
-TODO Add these 3 sections
+
+*TODO Add these 3 sections*
 
 ### Fix a typo
 
 ### Edit a chapter
 
 ### Add a chapter
 
+### Review a pull request
+
+Any pull request (PR) must be reviewed before it can be merged into the
+Cookbook. If you know someone who is familiar with the content you're
+proposing to add, request their review.
+
+The following notes are adapted from a coworking session in which Amy
+Steiker and Julie Lowndes reviewed a Cookbook [pull
+request](https://github.com/NASA-Openscapes/earthdata-cloud-cookbook/pull/171)
+from Andy Barrett. ([video](https://youtu.be/2_F5Kr6gM1I), 19 min). Some
+generic mechanics of reviewing a pull request in GitHub are demonstrated
+in a screenshare ([video](https://youtu.be/ZiG4-d5vT30), 7 min 41 sec).
+
+In reviewing a pull request with lots of changes, it can be helpful to
+have windows open to view both the GitHub browser and the Hub showing
+the Cookbook preview.
+
+From the pull request page in GitHub browser, look at the elements of
+the pull request.
+
+![](images/Screenshot%202023-07-10%20at%2012.55.55%20PM.png)
+
+-   Start with Conversation tab:
+    -   We can see all commits and comments on what Andy has worked on
+-   Commit tab:
+    -   More details on the commits that we saw under Conversation. When
+        we click on one of the commits, we can see line by line what has
+        changed under that commit (green lines are added, red lines have
+        been removed)
+-   Files Changed tab:
+    -   Often easier here to assess all the files that changed across
+        the commits
+    -   In Nav bar: Orange dot box signifies modified; Green plus box
+        means something's been added; Red minus box means deleted; Grey
+        arrow box means renamed.
+-   Checks tab:
+    -   Shows status of the Github Action that renders and deploys the
+        Cookbook. We can know whether this PR has the ability to be
+        deployed
+    -   This means two checks have passed
+
+Now, switch to the Hub to view the Cookbook preview as it would appear
+if the pull request was merged:
+
+-   Terminal:
+    -   Go to the Main branch and pull so that we have the most recent
+        changes from remote.
+    -   Git checkout to the branch that has the PR: reorg_how_to_guides
+        (note, this PR has been merged and the branch deleted so you
+        won't see this one; apply this example to your own)
+    -   Quarto preview - this will build the book with Andy's suggested
+        edits.
+        -   At the end of the preview log, it says "Browse at
+            https://openscapes.2i2c.cloud..." The Browse URL actually
+            points to the production Hub!! We need it to point to
+            staging since that's where we're working, so we use the URL
+            staging.openscapes.2i2c.cloud.... (See minute 10:06 in the
+            [video](https://youtu.be/2_F5Kr6gM1I?t=606).)
```

Comment:
I forget the reasoning behind this behavior. Do we want to mention that if you're running this from the production hub, then this is N/A?

Reply:
I've commented out that text because I did not see that behavior. We can bring it back and clarify if needed.